### PR TITLE
Add slack & github edit links to header

### PR DESCRIPTION
**Pull request by stevekrouse**
Now it looks like this:
Screen Shot 2022-02-15 at 2 57 57 PM

### CODE REVIEW

**Comment by stevekrouse** [comment]
Screen Shot 2022-02-15 at 3 50 54 PM

File: zaplib/docs/book.toml [hunk 1]
```diff
@@ -4,3 +4,11 @@ language = "en"
...
 src = "src"
 title = "Zaplib docs"
+
+[output.html]
+git-repository-url="https://github.com/Zaplib/zaplib"
+edit-url-template = "https://github.com/Zaplib/zaplib/edit/main/zaplib/docs/{path}"
+additional-css = ["theme/css/slack.css"]
+
+[output.html.print]
+enable = false
```
Comment (janpaul123) [inline zaplib/docs/book.toml:14]: Do you open VSCode workspace (`zaplib.code-workspace`)? It should have the setting `"files.insertFinalNewline": true,` in there which makes sure we don't get weird diffs at the end of files like this.

File: zaplib/docs/book.toml [hunk 2]
```diff
@@ -4,3 +4,11 @@ language = "en"
 multilingual = false
 src = "src"
 title = "Zaplib docs"
+
+[output.html]
+git-repository-url="https://github.com/Zaplib/zaplib"
+edit-url-template = "https://github.com/Zaplib/zaplib/edit/main/zaplib/docs/{path}"
+additional-css = ["theme/css/slack.css"]
```
Comment (janpaul123) [inline zaplib/docs/book.toml]: Rename to `custom.css` or something so we can add other things besides Slack stuff?

File: zaplib/docs/theme/index.hbs [hunk 3]
```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE HTML>
```
Comment (janpaul123) [inline zaplib/docs/theme/index.hbs:1]: Maybe add a comment at the top where you vendored this from (ideally with a git commit hash).

**Review by janpaul123 — APPROVED** [review]
Couple suggestions; looks good overall!

**Pull request merged**